staff/groups/aleti/project-app/src/logic/UserApi.js

```diff
@@ -0,0 +1,124 @@
+const UserApi = {
+
+    url: 'https://skylabcoders.herokuapp.com/api',
+    token: '',
+
+    loginUser(body){
+        return Promise.resolve()
+        .then(()=>{
+
+            if (!body) throw Error('Input body not valid')
+
+            if( typeof body.username !== 'string' || typeof body.password !== 'string' ) 
+                throw Error('Input body not valid')
+    
+            return  fetch(`${this.url}/auth`, { 
+                body: JSON.stringify(body),
+                headers: {
+                    "content-type": "application/json"
+                },
+                method: 'POST'
+            })
+            .then(resp => resp.json())
+            .then(info => info)
```

Comment:
WARN this then() does nothing here... it just adds a redundant link in the chain (promise then's)

Reply:
done!